core/src/main/java/org/infinispan/configuration/cache/Configurations.java

```diff
@@ -39,16 +39,6 @@ public static boolean isOnePhaseCommit(Configuration cfg) {
             cfg.transaction().lockingMode() == LockingMode.PESSIMISTIC;
    }
 
-   public static boolean isOnePhaseTotalOrderCommit(Configuration cfg) {
```

Comment:
Believe we have to deprecate this method first as it is in a public package. Should be able to just return false always I assume.

Reply:
Or maybe this method is obscure enough that we don't need to worry.

Reply:
hmmm.. I assumed this class was internal. I don't see any use case for a user to use any of this methods :/ 
If you want, I can add it back.

Reply:
It is in a package that is public. We should probably just put it back in and always return false and deprecate for now.

Reply:
done!